.github/workflows/build.yml

```diff
@@ -71,6 +71,15 @@ jobs:
         with:
           name: unit-test-results-${{matrix.os.name}}-${{github.sha}}
           path: build/test-report/unit
+      - name: Test Application Integration
+        id: 'test_integration'
+        run: npm run test:integration ${{ matrix.os.coverage }}
+      - name: Upload integration test results
+        if: ${{ failure() && steps.test_integration.outcome == 'failure' }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: integration-test-results-${{matrix.os.name}}-${{github.sha}}
+          path: build/test-report/e2e
```

Comment:
It's weird that the report for the integration tests are stored in **test-report/e2e** and not **test-report/integration**.

Reply:
hm, this looks like a mistake (copy/paste partially updated 😸). I am checking it

Reply:
Oops, I just saw your message now. I proposed a change after reading all the files ^^